src/rdpc/analysesProcessor.ts

```diff
@@ -39,8 +38,8 @@ export const analysisStream_withSpeicmens = async function* ({
   };
   analysesFetcher: typeof fetchAnalysesWithSpecimens;
   donorId?: string;
-}): AsyncGenerator<AnalysisWithSpecimens[]> {
-  const chunkSize = config?.chunkSize || 1000;
+}): AsyncGenerator<Analysis[]> {
+  let chunkSize = config?.chunkSize || 100;
```

Comment:
There is a default set in the Config file, so we don't need the || 100 anymore, unless a value of 100 is special to this scenario.